modules/microshift-install-rpms-low-latency.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-low-latency-rpm-package_{context}"]
+= Installing the {microshift-short} low latency RPM package
+
+When you install {microshift-short}, the low latency RPM package is not installed by default. You can install the low latency RPM as an optional package.
+//Can we also include this in a RHEL for Edge image?
+
+.Procedure
+
+. Install the low latency RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-low-latency
+----
++
+[TIP]
+====
+Restart the host after activating your TuneD profile. Restarting the host restarts {micoshift-short} and CRI-O, which applies the low latency manifests and activates the TuneD profile.
+====
+
+.Next steps
+. Configure the kubelet parameter for low latency in the {microshift-short} `config.yaml`.
+. Tune your operating system, for example, configure and activate a TuneD profile.
+. Optional: Configure automatic activation of your TuneD profile.
+. Optional: If you are using the x86_64 architecture, install {op-system-rt-kernel}.
+. Prepare your workloads for low latency.
+. Configure workload partitioning.
```

Comment:
I think we should move it up, this is part of crio & microshift configuration. Maybe they could 1st enable the WP, skip the MicroShift & CRIO restart, then proceed to low latency settings

Reply:
That does sound a lot better for the user. Revising.